builtin/core.wren

```diff
@@ -90,6 +90,14 @@ class Sequence {
 
     return result
   }
+
+  list {
+    var result = new List
```

Comment:
That's what I had before, but the Wren compiler not like the `[]` list constructor being used before the `List` class has been mentioned. But at that point it's too late to define `Sequence.list`. I then noticed all the other Sequence methods used `new List` so I used it as well.